roles/logging-config/handlers/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 - name: restart logstash-forwarder
   service: name=logstash-forwarder state=restarted
+  when: ansible_architecture != "ppc64le"
```

Comment:
Logstash binaries for ppc64le are not available in repos currently. It will be made available later.